tutorials/transfer_learning/transfer_learning.jl

```diff
@@ -48,13 +48,15 @@ include("dataloader.jl")
 # Finally, the model looks something like:
 
 model = Chain(
-  resnet[1:end-2],
+  resnet[1],
+  AdaptiveMeanPool((1, 1)),
+  Flux.flatten,
   Dense(2048, 1000),  
   Dense(1000, 256),
```

Comment:
It seems very odd to stack Dense layers with no nonlinearity. Would this still work?
```suggestion
  Dense(2048, 1000, relu),  
  Dense(1000, 256, relu),
```

Reply:
I also found it weird to stack linear layers but my goal was to get it working so that I had an example to base it off of and pushing the fix was a bonus.

It seems to work with the relu.